### DISCUSSION TITLE
While updating statamic to 3.4.* iam getting error for this plugin

### DISCUSSION THREAD

**Discussion by subramanya-idaksh** [duncanmcclean/cookie-notice#59] [Help]
While updating statamic to 3.4.* I am getting an error for this plugin
 Problem 1
    - doublethreedigital/cookie-notice is locked to version v7.1.1 and an update of this package was not requested.
    - doublethreedigital/cookie-notice v7.1.1 requires statamic/cms 3.2.* || 3.3.* -> found statamic/cms[v3.2.0-beta.1, ..., 3.3.x-dev] but it conflicts with your root 
composer.json require (3.4.*).

**Comment by duncanmcclean** [comment]
What does your `composer.json` file look like?

Also, what command are you running? Can you try `composer update` without any other parameters?

**Comment by subramanya-idaksh** [comment]
Hi, here I am attaching composer.lock file as well as composer.json file.
and I am using command composer update statamic/cms --with-dependencies

Thank you

**Reply by duncanmcclean** [reply]
Sorry, you haven't copied in your `composer.json` file (I don't need the lock file)..

**Comment by subramanya-idaksh** [comment]
Here I attach composer.json file

**Reply by duncanmcclean** [reply]
I still can't see an attachment. 

If your attaching something to the emails from GitHub, they won't show up on GitHub for me to see.
